linode/manager
# change: [M3-8022] - Update Rebuild Linode drawer

## Description 📝
Updates the Rebuild Linode drawer based on the latest UX mockups.

## Changes  🔄
- Update drawer title to match CTA ("Rebuild an Existing Linode from an Image")
- Display the selected image in the drawer
- Update select placeholder text to "Select Linode or Type to Search"
- Update action button text to "Rebuild Linode"

## Target release date 🗓️
6/24

## Preview 📷

| Before  | After   |
| ------- | ------- |
| ![Screenshot 2024-06-18 at 6 04 56 PM](https://github.com/linode/manager/assets/122488130/f37d4605-a18c-4054-8ed4-97c546f6c6f2)| ![Screenshot 2024-06-18 at 6 04 48 PM](https://github.com/linode/manager/assets/122488130/a772fe8f-3afc-4f87-954a-c0b10c203bad) |

## How to test 🧪

### Verification steps
- Activate the Rebuild Linode drawer (Images -> Action Menu (three dots) -> Rebuild an Existing Linode)
- Verify text and copy matches the mockup (see Figma link in the epic)
- Verify drawer functionality still works as expected (clicking the submit button will redirect to the Linode Rebuild dialog)
- Verify all CI and unit tests pass

## As an Author I have considered 🤔

*Check all that apply*
